dts/bindings/radio_fem/skyworks,sky66112-11.yaml

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+
+description: >
+    This is a representation of the Skyworks SKY66112-11 Radio Front-End module.
+
+compatible: "skyworks,sky66112-11"
+
+include: base.yaml
+
+properties:
+    ctx-pin:
+        type: int
+        description: >
+            Pin number of the SOC controlling CTX pin of the SKY66112-11
+
+    crx-pin:
+        type: int
```

Comment:
Wouldn't it be better to use here a `gpios` `phandle-array` type since you need for example information about pin polarity ?